src/components/Pressable/PressableWithDelayToggle.js

```diff
@@ -1,22 +1,19 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import * as Expensicons from '../Icon/Expensicons';
-import compose from '../../libs/compose';
 import Icon from '../Icon';
 import Tooltip from '../Tooltip';
 import Text from '../Text';
 import styles from '../../styles/styles';
 import variables from '../../styles/variables';
 import getButtonState from '../../libs/getButtonState';
 import * as StyleUtils from '../../styles/StyleUtils';
-import withLocalize, {withLocalizePropTypes} from '../withLocalize';
-import withDelayToggleButtonState, {withDelayToggleButtonStatePropTypes} from '../withDelayToggleButtonState';
 import PressableWithoutFeedback from './PressableWithoutFeedback';
+import useThrottledButtonState from '../../hooks/useThrottledButtonState';
 
 const propTypes = {
-    ...withLocalizePropTypes,
-
-    ...withDelayToggleButtonStatePropTypes,
+    /** Ref passed to the component by React.forwardRef (do not pass from parent) */
+    innerRef: PropTypes.oneOfType([PropTypes.func, PropTypes.shape({current: PropTypes.instanceOf(React.Component)})]).isRequired,
```

Comment:
My reasoning was that since it's always passed by `forwardRef`, then it would make sense to have it be required. However I see now that it will be null/undefined if the parent doesn't pass a ref.